src/math/lp/nla_intervals.cpp

```diff
@@ -302,72 +304,68 @@ bool intervals::interval_from_term(const nex& e, interval& i) {
 }
 
 template <e_with_deps wd>
-intervals::interval intervals::interval_of_sum_no_term(const nex_sum& e) {
-    const nex* inf_e = get_inf_interval_child(e);
-    if (inf_e) {
-        return interval();
-    }
-    interval a = interval_of_expr<wd>(e[0], 1);
+void intervals::interval_of_sum_no_term(const nex_sum& e, scoped_dep_interval & sdi) {
+    if (has_inf_interval(e))
+        return;
```

Comment:
would be good to set sdi to interval() explicitly.

Reply:
I can add an assert that makes sure the bounds are infinite.